internal/tiger/cmd/mcp_install.go

```diff
@@ -37,28 +37,44 @@ const (
 	KiroCLI     MCPClient = "kiro-cli"
 )
 
-// TigerMCPServer represents the Tiger MCP server configuration
-type TigerMCPServer struct {
+// MCPServerConfig represents the MCP server configuration
+type MCPServerConfig struct {
 	Command string   `json:"command"`
 	Args    []string `json:"args"`
 }
 
+// InstallOptions configures the MCP server installation behavior
+type InstallOptions struct {
+	// CreateBackup creates a backup of existing config files before modification (default: true for CLI)
+	CreateBackup bool
+	// CustomConfigPath overrides the default config file location
+	CustomConfigPath string
+	// ServerName is the name to register the MCP server as (default: "tiger")
+	ServerName string
```

Comment:
Is it true that this defaults to `tiger` (as it says in the comment)? I'm not sure it really should, if this is being exposed as a package that other projects can import and use.